nx-X11/config/cf/Imakefile

```diff
@@ -32,115 +31,24 @@ nxversion.def
 
 RULEFILES = \
 Imake.rules \
-Motif.rules \
-Win32.rules \
 X11.rules \
-bsdLib.rules \
```

Comment:
Deleting that... upon closer inspection, it turns out all these files have already been deleted anyway.